src/ssmDocument/commands/publishDocument.ts

```diff
@@ -119,28 +121,28 @@ export async function createDocument(
         outputChannel.appendLine(
             localize(
                 'AWS.message.info.ssmDocument.publishDocument.createSuccess',
-                "Successfully created and uploaded SSM Document '{0}'",
+                "Successfully created and uploaded Systems Manager Document '{0}'",
                 wizardResponse.name
             )
         )
         if (result.DocumentDescription) {
             outputChannel.appendLine(stringify(result.DocumentDescription))
         }
-        logger.info(`Created SSM Document successfully ${stringify(result.DocumentDescription)}`)
-        vscode.window.showInformationMessage(`Created SSM Document successfully`)
+        logger.info(`Created Systems Manager Document successfully ${stringify(result.DocumentDescription)}`)
+        vscode.window.showInformationMessage(`Created Systems Manager Document successfully`)
         outputChannel.appendLine('')
     } catch (err) {
-        logger.info(`Failed to create SSM Document '${wizardResponse.name}'. %0`, err as Error)
+        logger.info(`Failed to create Systems Manager Document '${wizardResponse.name}'. %0`, err as Error)
```

Comment:
Seeing this in a few places throughout this file (might be elsewhere too): should the `logger` statements in the catch blocks be `logger.error` instead of `logger.info`?

Reply:
Will change to logger.error for catch blocks